files/build_templates/docker_image_ctl.j2

```diff
@@ -92,6 +88,7 @@ function preStartAction()
 function postStartAction()
 {
 {%- if docker_container_name == "database" %}
+    WARM_DIR=/host/warmboot
```

Comment:
>WARM_DIR [](start = 4, length = 8)

Why add here?

Reply:
If this is a true bug, move it to file global?

---
In reply to: [535743785](https://github.com/Azure/sonic-buildimage/pull/5773#discussion_r535743785) [](ancestors = 535743785)

Reply:
It is not a true BUG, WARM_DIR is set as global when preStart is invoked first and postStart can use its value later.